src/tms_service.ts

```diff
@@ -220,7 +220,7 @@ export class TMSService extends AbstractEmsService {
   */
   public static transformColorProperties(
     layer: LayerSpecification,
-    color: string,
+    color: string | undefined,
     operation: blendMode,
```

Comment:
We could also do this, since `colorizeColor` has a default operation argument.

```suggestion
    color?: string,
    operation?: blendMode,
```

Reply:
Correct, thanks this makes the added test even more succinct.